docs/protocol/ISM/aggregation-ISM.mdx

```diff
@@ -14,4 +30,4 @@ Developers simply need to implement the `modulesAndThreshold()` function.
 
 By creating a custom implementation, application developers can tailor the security provided by a `AggregationISM` to the needs of their application.
 
-For example, a custom implementation could require that low value messages be verified by a [Multisig ISM](multisig-ISM.mdx), and require that high value messages _also_ be verified by a [Wormhole ISM](./third-party-ISMs/wormhole-ISM.mdx).
+For example, a custom implementation could require that low value messages be verified by a [Multisig ISM](./multisig-ISM.mdx), and require that high value messages _also_ be verified by a Wormhole ISM.
```

Comment:
Since the wormhole ism is wip in removed it. I think it can create a confusion that its ready to use, and if someone clicks and follows the link they won't be able to use it atm. Imo we can add more use-cases for isms, but link to the ones that are ready to use rn. 

lmk what you think? ı can add it again if you feel more strongly towards having it there.  

![Screenshot 2024-10-18 at 11 46 07](https://github.com/user-attachments/assets/fa683aa7-2825-4f3d-b817-4dec6a6094cc)

Reply:
that's fine with me - happy to cut it!